src/elements/codelabs-card.html

```diff
@@ -22,9 +23,13 @@
         font-family: 'Ubuntu',Helvetica,Arial;
         height: 100%;
         position: relative;
+
+        --paper-card-content: {
+          padding: 0 16px 16px 16px;
+        };
       }
       paper-card a {
-        color: var(--primary-text-color);
+        color: #111;
```

Comment:
Is this intentionally different from the primary text color, or should it be updated in `ubuntu-tutorials-app.html`?